packages/server/src/helpers/decodeAuthenticatorExtensions.ts

```diff
@@ -0,0 +1,37 @@
+import cbor from 'cbor';
+
+/**
+ * Convert authenticator extension data buffer to a proper object
+ *
+ * @param extensionData Authenticator Extension Data buffer
+ */
+export default function decodeAuthenticatorExtensionData(
+  extensionData: Buffer
+): AuthenticationExtensionsAuthenticatorOutputs | undefined {
+  let toCBOR: AuthenticationExtensionsAuthenticatorOutputs | undefined;
+  try {
+    toCBOR = cbor.decodeAllSync(extensionData)[0];
+  } catch (e) {
+    console.error(e);
+    toCBOR = undefined;
+  }
```

Comment:
Looking at existing use of `cbor.decodeAllSync(...)[0]`, I think I want this method to go the way of `convertPublicKeyToPEM()`, which throws on bad data but tries to provide some context. For consistency's sake, how about this instead?

```suggestion
  try {
    toCBOR = cbor.decodeAllSync(extensionData)[0];
  } catch (err) {
    const _err = err as Error;
    throw new Error(`Error decoding authenticator extensions: ${_err.message}`);
  }
```

Reply:
I've already committed this, but throwing means it needs to be caught by the caller. Do you think that's the way to go? I'd just skip parsing rather than terminating entire verification process. What do you think?

Reply:
My current thinking is that if we're told that extension data is present (via `flags.ed`) but it's malformed (improper CBOR, etc...) then I think it makes sense to terminate since it indicates an issue with the authenticator.

Reply:
I think that makes sense!